e2e/tests/negative/upgrade_conflict_handle.robot

```diff
@@ -0,0 +1,95 @@
+*** Settings ***
+Documentation    Manual Test Cases
+Test Tags    negative
+
+Resource    ../keywords/variables.resource
+Resource    ../keywords/common.resource
+Resource    ../keywords/longhorn.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/storageclass.resource
+Resource    ../keywords/workload.resource
+Resource    ../keywords/statefulset.resource
+Resource    ../keywords/backup.resource
+Resource    ../keywords/recurringjob.resource
+Library     DateTime
+
+Test Setup    Set up test environment
+Test Teardown    Cleanup test resources
+
+*** Keywords ***
+Get test start time
+    ${test_start_time}=    Get Current Date    result_format=datetime
+    Log    ${test_start_time}
+    Set Test Variable    ${test_start_time}
+
+*** Test Cases ***
+Test Stability of Longhorn with Large Workload
+    ${NUM_VOLUMES} =    Set Variable    100
+    ${NUM_VOLUMES_DETACH} =    Set Variable    10
+
+    Given Set setting deleting-confirmation-flag to true
+    And Get test start time
+    
+    When Create storageclass longhorn-test with    dataEngine=${DATA_ENGINE}
+    FOR    ${i}    IN RANGE    ${NUM_VOLUMES}
+        And Create statefulset ${i} using RWO volume with longhorn-test storageclass and size 10 Mi
+        And Write 1 MB data to file data.txt in statefulset ${i}
+    END
+
+    And Create snapshot recurringjob 0
+    ...    groups=["default"]
+    ...    cron=*/1 * * * *
+    ...    concurrency=5
+    ...    labels={"test":"recurringjob"}
+
+    FOR    ${i}    IN RANGE    ${NUM_VOLUMES_DETACH}
```

Comment:
Could I know how long will this for loop take?

Since the original test case has an observation period for recurring jobs:
```
6. Observe the setup for 1/2 an hr.
```

Reply:
Ten volumes consume about 15 minutes, to reach 30 minutes, `NUM_VOLUMES_DETACH` changed to 20